android/app/src/main/java/com/dismoi/scout/accessibility/BackgroundService.kt

```diff
@@ -135,54 +128,56 @@ class BackgroundService : AccessibilityService() {
       return
     }
 
-    val parentNodeInfo: AccessibilityNodeInfo = event.source ?: return
-
-    if (overlayIsActivated(applicationContext) && isWindowChangeEvent(event)) {
+    if (overlayIsActivated(applicationContext)) {
       val packageName = event.packageName.toString()
 
-      if (isLauncherPackage(packageName) && parentNodeInfo.className.toString() != "android.widget.FrameLayout") {
-        _hide = "true"
-        handler.post(runnableCode)
-        return
+      if (getEventType(event) == "TYPE_WINDOW_STATE_CHANGED" && packageName != "com.android.chrome") {
```

Comment:
Why don’t we use BrowserConfig anymore ?

Reply:
It's used and it's here no ? https://github.com/dis-moi/mobile_poc/tree/master/android/app/src/main/java/com/dismoi/scout/accessibility/browser

Reply:
OK but I mean it’s pointless to have `SupportedBrowserConfig` if we just throw some Chrome specific code in here, don’t you think ?

Reply:
Was thinking of using a Firefox class as a children of Browser but maybe it's a bit premature indeed. What do you propose? Example?

Reply:
I suggest that anything browser specific is contained in a `SupportedBrowserConfig`, as in the beginning, so that the service only finds the current browser and applies the given config, as you did with `addressBarId` …